public/javascripts/global.js

```diff
@@ -129,14 +129,15 @@ function fetchQuiz(event) {
     var target = $(event.target);
     if (target.hasClass('quiz-btn')){
       var team = $('.selected-team'),
-          id = team.data('id');
+          id = team.data('team-id'),
```

Comment:
I think that makes sense for now. I'm still leery of being tied to the API ID. If a record of all of the teams is in mongo (which they should be, no?) wouldn't it be better to key off of the RB id, and then fetch an update from the API depending on our rules?